quarkus/admin/build.gradle.kts

```diff
@@ -75,3 +75,13 @@ publishing {
     }
   }
 }
+
+tasks.named("compileJava") { dependsOn("compileQuarkusGeneratedSourcesJava") }
+
+tasks.named("sourcesJar") {
+  dependsOn("compileQuarkusGeneratedSourcesJava")
+}
+
+// Add when there's code built in this module
```

Comment:
Hm. There _is_ code built in this module. I guess you meant instead: when this module will be imported by other Quarkus module (IOW when the jandex plugin will be added)?

Reply:
argh - the jandex plugin was missing :facepalm: